relay-cardinality/src/config.rs

```diff
@@ -53,6 +53,12 @@ pub enum CardinalityScope {
     /// The limit will be enforced for a specific project.
     Project,
 
+    /// A per metric type cardinality limit.
+    ///
+    /// This limit usually doesn't make sense to enforce,
+    /// but is useful for cardinality reports.
+    Type,
```

Comment:
Is this per org, project _and_ type? Might be worth documenting (i.e. not a per-type limit across projects of an org).

Reply:
Thanks, tried to improve the docs, let me know what you think.